proposals/lip-0018.md

```diff
@@ -190,7 +190,7 @@ The type of the reclaim transaction is the lowest positive integer that is not u
 
 ###### Schema
 
-The `asset` property of a reclaim transaction needs to contain the `amount` property which is of type _string_. The value of `asset.amount` must represent a decimal number and this number must be equal to the balance associated to the address of the current format in Beddows.
+The `asset` property of a reclaim transaction needs to contain the `amount` property which is of type _uint64_. The value of `asset.amount` must represent a number and this number must be equal to the balance associated to the address of the current format in Beddows.
```

Comment:
```suggestion
The `asset` property of a reclaim transaction needs to contain the `amount` property which is of type _uint64_. The value of `asset.amount` must be equal to the balance associated to the address of the current format in Beddows.
```

I think that specifying that it must represent a number is redundant now (that is type uint64)